Project.toml

```
@@ -40,8 +39,6 @@ Polymake = "0.11.6"
 Preferences = "1"
 Random = "1.6"
 RandomExtensions = "0.4.3"
-RecipesBase = "1.2.1"
-Serialization = "1.6"
 Singular = "0.18.17"
```

```suggestion
Serialization = "1.6"
Singular = "0.18.17"
```
Please don't remove the compat entry for Serialization.